src/RSCoin/Mintette/Launcher.hs

```diff
@@ -46,6 +48,13 @@ launchMintetteReal deleteIfExists epochDelta port env dbPath ctxArg =
         fork_ $ runWorkerWithDelta epochDelta env st
         serve port st env
 
+addToBank :: Bool -> FilePath -> ContextArgument -> SecretKey -> String -> Int -> IO ()
```

Comment:
This function should't take first two arguments, because it doesn't use db at all. `mintetteWrapperReal` is not needed here, because it assumes that `State` will be used. You just need to use `runRealModeUntrusted` directly.